website/blog/2020-04-22-deployments-made-better.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "Deployment Experience made better!"
+author: Subodh Kumar
+---
+
+Deployment is a process which enables the developers to serve the apps to users. 
+With the latest release, WaveMaker has made the discovery of deployment failures in the application fast & easy.
```

Comment:
When user invokes Deploy operation WaveMaker trigger the maven command to build the project and generate artefacts for deployment. Build process includes angular build process to generated optimized javascript assets. During the build process there can be failures.

Reply:
Added.